docs/modules/reference/pages/configuration/performance-data-collection/introduction.adoc

```diff
@@ -0,0 +1,6 @@
+[[ref-performance-data-collectors]]
+= Collectors
+
+This section describes all available built-in performance data collectors.
+You can configure them to allow complex monitoring to meet the needs of your environment.
```

Comment:
Since this moved out of Operation, there should be a link here for access to Metadata info.
```suggestion
You can configure them to allow complex monitoring to meet the needs of your environment.
You can use xref:operation:meta-data.adoc#ga-meta-data-dsl[Metadata-DSL] to dynamically interpolate metadata into parameter values during the collection process.
```